packages/offix-client/src/config/ApolloOfflineClientConfigValidator.ts

```diff
@@ -0,0 +1,17 @@
+import { ApolloOfflineClientConfig } from "./ApolloOfflineClientConfig";
+import { ConfigError } from "./ConfigError";
+
+export default function(config: ApolloOfflineClientConfig) {
+    checkFields(config, [
+        "conflictStrategy", "conflictProvider", "cacheStorage",
+        "offlineStorage", "conflictListener", "cache"
+    ]);
+}
+
+function checkFields(obj: any, fields: string[]) {
+    fields.forEach((fieldName) => {
+        if (obj[fieldName] === undefined) {
+            throw new ConfigError(`${fieldName} is required`, fieldName);
```

Comment:
This is just nitpicking, but can we get a slightly more meaningful message here, something like:
`Offix client config error: ${fieldName} is required`